### PR TITLE
Fix website in StructuredDataSpider

### PR DESCRIPTION
**Pull request by Cj-Malone**
This used to be set by LinkedDataParser.parse but that's no longer used.

### CODE REVIEW

File: locations/linked_data_parser.py [hunk 1]
```diff
@@ -161,11 +161,10 @@ def parse(response, wanted_type, parse_json5=False) -> GeojsonPointItem:
         if ld_item:
             item = LinkedDataParser.parse_ld(ld_item)
 
-            if item["website"] is None:
-                item["website"] = response.url
-            elif item["website"] == "":
-                item["website"] = response.url
-            elif item["website"][0] == "/":
+            if isinstance(item["website"], list):
```
Comment (iandees) [inline locations/linked_data_parser.py:164]: Check that there's something in the list before indexing into it:

```suggestion
            if isinstance(item["website"], list) and item["website"]:
```

File: locations/linked_data_parser.py [hunk 2]
```diff
@@ -161,11 +161,10 @@ def parse(response, wanted_type, parse_json5=False) -> GeojsonPointItem:
...
+                item["website"] = item["website"][0]
+
+            if item["website"] is None or item["website"] == "":
```
Comment (iandees) [inline locations/linked_data_parser.py:167]: Same as:

```suggestion
            if not item["website"]:
```

File: locations/structured_data_spider.py [hunk 3]
```diff
@@ -85,6 +85,15 @@ def parse_sd(self, response):
                     if item["ref"] is None:
                         item["ref"] = response.url
 
+                item["website"] = None
+                if isinstance(item["website"], list):
```
Comment (iandees) [inline locations/structured_data_spider.py:89]:
```suggestion
                if isinstance(item["website"], list) and item["website"]:
```

File: locations/structured_data_spider.py [hunk 4]
```diff
@@ -85,6 +85,15 @@ def parse_sd(self, response):
...
+                    item["website"] = item["website"][0]
+
+                if item["website"] is None or item["website"] == "":
```
Comment (iandees) [inline locations/structured_data_spider.py:92]:
```suggestion
                if not item["website"]:
```

File: locations/linked_data_parser.py (hunk 1 above)
Comment (Cj-Malone) [reply locations/linked_data_parser.py:164]: `parse_ld` always populates `website`, even if it's None. `isinstance(None, list)` works as expected. Can update if you want though.

Comment (iandees) [reply locations/linked_data_parser.py:164]: No prob.

**Pull request merged**